### ISSUE TITLE
Binance blocking ip when run multiple order pure market making. 

### ISSUE THREAD

**Issue by frndxyz** [CoinAlpha/hummingbot#1031]

**Comment by BUDDHA-TONY** [comment]
You may need to provide certain informations such as logs/conf and a screenshot of your terminal when the error happened. 
Please follow the guidelines when your trying to create a github bug issue.

```
**Describe the bug**
// A clear and concise description of what the bug is.

**Steps To Reproduce**
// A concise description of the **steps to reproduce the buggy behavior**:

Here are the steps to reproduce the issue (see attachments in the section below):
1. Set up your configuration file with these parameters: ...
2. Start the bot using this command: ...
...

**Screenshots**
// If applicable, add screenshots to help explain your problem.

**Release version**
// Include your bot's version number (Can be found at the bottom left corner of your CLI).

**Attachments**
// Please attach your config file and log file to help us debug. It would be difficult for us to help you without those! 


*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:**
**Optional: your ETH address:** (If you want to participate in the bug bounty)
```

**Comment by frndxyz** [comment]
i have attached this log and strategy files.

Bot version: 0.17.1

unable to get screenshot because later i stopped bot.


thanks

**Comment by frndxyz** [comment]
unable to attach file here.

**Comment by frndxyz** [comment]
Got error again 

![49A6EAE6-CF58-4BFC-AC43-55B3B641514D](https://user-images.githubusercontent.com/6427428/68203132-297d8480-ffeb-11e9-8ce9-eec52e522490.jpeg)
![B3865542-659A-42BA-AE47-4C3C8F6C156E](https://user-images.githubusercontent.com/6427428/68203135-2a161b00-ffeb-11e9-8f2b-015b0c8ea411.jpeg)

**Comment by frndxyz** [comment]
My config 

![2DF77DBC-9F47-441C-B89E-53122C330300](https://user-images.githubusercontent.com/6427428/68203439-cdffc680-ffeb-11e9-8645-4f18c520e31e.jpeg)

**Comment by PtrckM** [comment]
IP banned should take from minutes to an hour, if you are doing some testing with the bot we suggest to turn on paper_trade.

I run your PMM config on latest release 0.18.0 no error

**Comment by PtrckM** [comment]
Please attach the logs as well [here how](https://help.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests)

**Comment by frndxyz** [comment]
here is log file.

[logs_conf_pure_market_making_strategy_0.log](https://github.com/CoinAlpha/hummingbot/files/3822947/logs_conf_pure_market_making_strategy_0.log)

**Comment by PtrckM** [comment]
Please try updating the bot to 0.18.1 latest version, see if problem still occurs.

**Comment by BUDDHA-TONY** [comment]
I reached out to binance and I was informed that it will be less than 24hours for the ban to be lifted.

**Comment by frndxyz** [comment]
> I reached out to binance and I was informed that it will be less than 24hours for the ban to be lifted.

But that again got ban when using multiple orders.

**Comment by frndxyz** [comment]
> In the meantime try updating the bot to 0.18.1 latest version, see if problem still occurs.

I will try it

**Comment by bb111189** [comment]
faced similar issue on 0.20.0

**Comment by mifeng** [comment]
Fixed by #1673
